Makefile

```diff
@@ -12,28 +13,37 @@ build: core $(all_apps)
 .PHONY: clean
 clean: clean-core $(addprefix clean-app-,$(apps))
 
+node_modules: package.json package-lock.json
```

Comment:
this rule tells "the target of this build is the 'node_modules' folder". so if this folder does not exist it will run it.

also, it will run it if the mtime of "package.json" or "package-lock.json" is more recent than the target "node_modules". so every time you change one of these, it will rerun this rule.